apps/src/gamelab/GameLabView.jsx

```diff
@@ -8,6 +9,25 @@ var GameLabInterfaceMode = require('./constants').GameLabInterfaceMode;
 var GameLabVisualizationHeader = require('./GameLabVisualizationHeader.jsx');
 var ProtectedStatefulDiv = require('../templates/ProtectedStatefulDiv.jsx');
 
+var styles = {
+  codeWorkspace: {
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    left: 400,
+    bottom: 0,
+    marginLeft: 15,
+    border: '1px solid #ddd',
+    overflow: 'hidden',
+  },
+  codeWorkspaceRTL: {
+    right: 400,
+    left: 0,
+    marginRight: 15,
+    marginLeft: 0
```

Comment:
At this point, it might be easy to extract some common constants from these two styles where they're obviously related, like the left:400/right:400 switch.